testing/upload_validation.ts

```diff
@@ -772,6 +780,22 @@ function validateFormulas(schema: z.ZodObject<any>) {
     });
 }
 
+function validateFormatMatchers(values: string[]): boolean {
```

Comment:
Hmm, I think this should be accepting a string and not an array now, because it's getting called once per matcher now, instead of once for all the matchers together.

Reply:
Ah, I had put the `.refine` outside the `.array()` call. Fixed.